mlir/lib/CAPI/ExecutionEngine/CMakeLists.txt

```diff
@@ -7,6 +7,10 @@ set(LLVM_LINK_COMPONENTS
 add_mlir_upstream_c_api_library(MLIRCAPIExecutionEngine
   ExecutionEngine.cpp
 
+  LINK_COMPONENTS
+  OrcJit
+  Support
```

Comment:
I'd either add these to LLVM_LINK_COMPONENTS above or move the LLVM_LINK_COMPONENTS here as well. Odd to split them this way.

Reply:
Sorry about that. Doing too many things simultaneously.